### DISCUSSION TITLE
Shouldn't this work?

### DISCUSSION THREAD

**Discussion by RJ-Make** [nextcloud/all-in-one#4801] [Questions] [answered]
I'm currently running a Hansson-It Nextcloud in a Hyper-V using desec.io DNS for access.  Fully working and functional, no problem accessing.

Shouldn't I be able to simply change my router's Port Forward IP FROM the VM IP, to the Machine IP (same machine) that Docker Desktop is installed? (Ports, 80,443,8443 open on the router and windows firewall)

https://192.168.1.26:8080 of course works fine to get the AIO page and all containers are started and identified as 'green'. 

1) I cant pass the domain check.
2) If I add the "bypass..", I get a CERT error. 
2) I've tried https://your-domain-that-points-to-this-server.tld:8443 returns 'Server Not Found'

**Comment by szaimen** [comment]
> 2\. If I add the "bypass..", I get a CERT error.

What exact error do you get if you do so?

**Comment by RJ-Make** [comment]
So I completely removed everything and started over again, I get past the domain check now but end up with the same error. 

![2024-06-10_7-30-18](https://github.com/nextcloud/all-in-one/assets/8496835/fbca3a90-b099-4c62-84b6-2ddda66ef95b)

I see this log entry in the NextCloud container logs. 

```
Failed to fetch discovery endpoint from https://XXXXX.XXXXX.io/
cURL error 35: OpenSSL/3.1.5: error:0A000438:SSL routines::tlsv1 alert internal error (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://XXXXX.XXXXX.io/hosting/discover
```

This still fails.

https://your-domain-that-points-to-this-server.tld:8443

**Comment by szaimen** [comment]
I see. Can you follow https://github.com/nextcloud/all-in-one/discussions/2105?

**Comment by RJ-Make** [comment]
Here are my Apache logs.

1) I have a single A record pointing to my machine's IP
2) I have a NS record with desec' name servers.
3) I'm not using any proxy
4) 443 Forwards correctly. 
5) Here is what I've selected on installation. (if it helps)

![Snag_135a9c01](https://github.com/nextcloud/all-in-one/assets/8496835/0d5ce992-acac-4b6e-9a24-cd5dcd33638d)

```
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
Connection to nextcloud-aio-nextcloud (172.20.0.10) 9000 port [tcp/*] succeeded!
{"level":"info","ts":1718021223.1553555,"msg":"using provided configuration","config_file":"/tmp/Caddyfile","config_adapter":""}
{"level":"info","ts":1718021223.1652305,"msg":"failed to sufficiently increase receive buffer size (was: 208 kiB, wanted: 2048 kiB, got: 416 kiB). See https://github.com/quic-go/quic-go/wiki/UDP-Buffer-Sizes for details."}
{"level":"info","ts":1718021223.1686153,"msg":"[INFO][FileStorage:/mnt/data/caddy] Lock for 'issue_cert_xxx.xx.xxx.xx' is stale (created: 2024-06-10 07:36:00.290727978 -0400 EDT, last update: 2024-06-10 07:45:01.229985887 -0400 EDT); removing then retrying: /mnt/data/caddy/locks/issue_cert_xxx.xx.xxx.xx.lock"}
[Mon Jun 10 08:07:03.185935 2024] [mpm_event:notice] [pid 55:tid 140338393549576] AH00489: Apache/2.4.59 (Unix) configured -- resuming normal operations
[Mon Jun 10 08:07:03.186122 2024] [core:notice] [pid 55:tid 140338393549576] AH00094: Command line: '/usr/local/apache2/bin/httpd -D FOREGROUND'
{"level":"error","ts":1718021234.2177813,"logger":"tls.issuance.acme.acme_client","msg":"challenge failed","identifier":"xxx.xx.xxx.xx","challenge_type":"tls-alpn-01","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]}}
{"level":"error","ts":1718021234.2178032,"logger":"tls.issuance.acme.acme_client","msg":"validating authorization","identifier":"xxx.xx.xxx.xx","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]},"order":"https://acme-v02.api.letsencrypt.org/acme/order/1774073037/277116237427","attempt":1,"max_attempts":3}
{"level":"error","ts":1718021234.2178178,"logger":"tls.obtain","msg":"could not get certificate from issuer","identifier":"xxx.xx.xxx.xx","issuer":"acme-v02.api.letsencrypt.org-directory","error":"HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)"}
{"level":"error","ts":1718021234.2178354,"logger":"tls.obtain","msg":"will retry","error":"[xxx.xx.xxx.xx] Obtain: [xxx.xx.xxx.xx] solving challenge: xxx.xx.xxx.xx: [xxx.xx.xxx.xx] authorization failed: HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem) (ca=https://acme-v02.api.letsencrypt.org/directory)","attempt":1,"retrying_in":60,"elapsed":11.04130646,"max_duration":2592000}
{"level":"error","ts":1718021305.2944057,"logger":"tls.issuance.acme.acme_client","msg":"challenge failed","identifier":"xxx.xx.xxx.xx","challenge_type":"tls-alpn-01","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]}}
{"level":"error","ts":1718021305.2944527,"logger":"tls.issuance.acme.acme_client","msg":"validating authorization","identifier":"xxx.xx.xxx.xx","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]},"order":"https://acme-staging-v02.api.letsencrypt.org/acme/order/151495713/17105190533","attempt":1,"max_attempts":3}
{"level":"error","ts":1718021305.2944934,"logger":"tls.obtain","msg":"could not get certificate from issuer","identifier":"xxx.xx.xxx.xx","issuer":"acme-v02.api.letsencrypt.org-directory","error":"HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)"}
{"level":"error","ts":1718021305.294508,"logger":"tls.obtain","msg":"will retry","error":"[xxx.xx.xxx.xx] Obtain: [xxx.xx.xxx.xx] solving challenge: xxx.xx.xxx.xx: [xxx.xx.xxx.xx] authorization failed: HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem) (ca=https://acme-staging-v02.api.letsencrypt.org/directory)","attempt":2,"retrying_in":120,"elapsed":82.122759287,"max_duration":2592000}
{"level":"error","ts":1718021436.1365576,"logger":"tls.issuance.acme.acme_client","msg":"challenge failed","identifier":"xxx.xx.xxx.xx","challenge_type":"tls-alpn-01","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]}}
{"level":"error","ts":1718021436.1366057,"logger":"tls.issuance.acme.acme_client","msg":"validating authorization","identifier":"xxx.xx.xxx.xx","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]},"order":"https://acme-staging-v02.api.letsencrypt.org/acme/order/151495713/17105231643","attempt":1,"max_attempts":3}
{"level":"error","ts":1718021436.1366274,"logger":"tls.obtain","msg":"could not get certificate from issuer","identifier":"xxx.xx.xxx.xx","issuer":"acme-v02.api.letsencrypt.org-directory","error":"HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)"}
{"level":"error","ts":1718021436.136653,"logger":"tls.obtain","msg":"will retry","error":"[xxx.xx.xxx.xx] Obtain: [xxx.xx.xxx.xx] solving challenge: xxx.xx.xxx.xx: [xxx.xx.xxx.xx] authorization failed: HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem) (ca=https://acme-staging-v02.api.letsencrypt.org/directory)","attempt":3,"retrying_in":120,"elapsed":212.972014677,"max_duration":2592000}
```

**Reply by szaimen** [reply]
> {"level":"error","ts":1718021234.2177813,"logger":"tls.issuance.acme.acme_client","msg":"challenge failed","identifier":"xxx.xx.xxx.xx","challenge_type":"tls-alpn-01","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]}}
> {"level":"error","ts":1718021234.2178032,"logger":"tls.issuance.acme.acme_client","msg":"validating authorization","identifier":"xxx.xx.xxx.xx","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]},"order":"https://acme-v02.api.letsencrypt.org/acme/order/1774073037/277116237427","attempt":1,"max_attempts":3}
> {"level":"error","ts":1718021234.2178178,"logger":"tls.obtain","msg":"could not get certificate from issuer","identifier":"xxx.xx.xxx.xx","issuer":"acme-v02.api.letsencrypt.org-directory","error":"HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)"}
> {"level":"error","ts":1718021234.2178354,"logger":"tls.obtain","msg":"will retry","error":"[xxx.xx.xxx.xx] Obtain: [xxx.xx.xxx.xx] solving challenge: xxx.xx.xxx.xx: [xxx.xx.xxx.xx] authorization failed: HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem) (ca=https://acme-v02.api.letsencrypt.org/directory)","attempt":1,"retrying_in":60,"elapsed":11.04130646,"max_duration":2592000}
> {"level":"error","ts":1718021305.2944057,"logger":"tls.issuance.acme.acme_client","msg":"challenge failed","identifier":"xxx.xx.xxx.xx","challenge_type":"tls-alpn-01","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]}}
> {"level":"error","ts":1718021305.2944527,"logger":"tls.issuance.acme.acme_client","msg":"validating authorization","identifier":"xxx.xx.xxx.xx","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]},"order":"https://acme-staging-v02.api.letsencrypt.org/acme/order/151495713/17105190533","attempt":1,"max_attempts":3}
> {"level":"error","ts":1718021305.2944934,"logger":"tls.obtain","msg":"could not get certificate from issuer","identifier":"xxx.xx.xxx.xx","issuer":"acme-v02.api.letsencrypt.org-directory","error":"HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)"}
> {"level":"error","ts":1718021305.294508,"logger":"tls.obtain","msg":"will retry","error":"[xxx.xx.xxx.xx] Obtain: [xxx.xx.xxx.xx] solving challenge: xxx.xx.xxx.xx: [xxx.xx.xxx.xx] authorization failed: HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem) (ca=https://acme-staging-v02.api.letsencrypt.org/directory)","attempt":2,"retrying_in":120,"elapsed":82.122759287,"max_duration":2592000}
> {"level":"error","ts":1718021436.1365576,"logger":"tls.issuance.acme.acme_client","msg":"challenge failed","identifier":"xxx.xx.xxx.xx","challenge_type":"tls-alpn-01","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]}}
> {"level":"error","ts":1718021436.1366057,"logger":"tls.issuance.acme.acme_client","msg":"validating authorization","identifier":"xxx.xx.xxx.xx","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]},"order":"https://acme-staging-v02.api.letsencrypt.org/acme/order/151495713/17105231643","attempt":1,"max_attempts":3}
> {"level":"error","ts":1718021436.1366274,"logger":"tls.obtain","msg":"could not get certificate from issuer","identifier":"xxx.xx.xxx.xx","issuer":"acme-v02.api.letsencrypt.org-directory","error":"HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)"}
> {"level":"error","ts":1718021436.136653,"logger":"tls.obtain","msg":"will retry","error":"[xxx.xx.xxx.xx] Obtain: [xxx.xx.xxx.xx] solving challenge: xxx.xx.xxx.xx: [xxx.xx.xxx.xx] authorization failed: HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem) (ca=https://acme-staging-v02.api.letsencrypt.org/directory)","attempt":3,"retrying_in":120,"elapsed":212.972014677,"max_duration":2592000}

This is your problem. Is the ip-address correctly pointing at you public ip-address? and is port 443 correclty forwarded to that server?

**Reply by RJ-Make** [reply]
Yes to both questions. 

My existing Hyper-V Nextcloud VM connects with no problem. The only thing I change is pointing the IP from my VM to my Machine IP that docker is installed in my router. 

I didn't think the domain check would be successful if those 2 were not correct??

**Reply by szaimen** [reply]
> I didn't think the domain check would be successful if those 2 were not correct??

Ah so you didn't skip the domain validation by setting `SKIP_DOMAIN_VALIDATION=true` ?

**Reply by RJ-Make** [reply]
That is correct, on my 2nd complete removal and re-attempt I left that at `SKIP_DOMAIN_VALIDATION=false`.

**Reply by szaimen** [reply]
Hm... Can you check if you can access the domain from outside your home network eg via a mobile connection?

**Reply by RJ-Make** [reply]
It did not work. This is very strange. Considering the Hyper-V VM has no problem connecting  (I also have access with a VMWare (Home Assistant) VM), I have to believe that this is either a Docker or AIO issue. 

I don't know how all this works but is it possible the desec DNS service is locked to a specific program?

This is what CurrPorts shows after I hit the Start Containers button. The Unknown entries keep showing up and then disappear over and over again. 

They all go away after I shutdown the containers. I also don't see any 443 entries when I start the Hyper-V VM back up either.

```
com.docker.backend.exe	1052	UDP	443	https	0.0.0.0										C:\Program Files\Docker\Docker\resources\com.docker.backend.exe	Docker Desktop Backend	Docker Desktop Backend	4.31.0.153195	Docker Inc.	6/10/2024 8:37:26 AM	SERVER\USER		A	6/11/2024 5:59:45 PM	6/11/2024 5:59:45 PM						
com.docker.backend.exe	1052	UDP	443	https	::				SERVER						C:\Program Files\Docker\Docker\resources\com.docker.backend.exe	Docker Desktop Backend	Docker Desktop Backend	4.31.0.153195	Docker Inc.	6/10/2024 8:37:26 AM	SERVER\USER		A	6/11/2024 5:59:45 PM	6/11/2024 5:59:45 PM						
com.docker.backend.exe	1052	TCP	443	https	0.0.0.0			0.0.0.0		Listening					C:\Program Files\Docker\Docker\resources\com.docker.backend.exe	Docker Desktop Backend	Docker Desktop Backend	4.31.0.153195	Docker Inc.	6/10/2024 8:37:26 AM	SERVER\USER		A	6/11/2024 5:59:47 PM	6/11/2024 5:59:45 PM						
com.docker.backend.exe	1052	TCP	443	https	::			::	SERVER	Listening					C:\Program Files\Docker\Docker\resources\com.docker.backend.exe	Docker Desktop Backend	Docker Desktop Backend	4.31.0.153195	Docker Inc.	6/10/2024 8:37:26 AM	SERVER\USER		A	6/11/2024 5:59:47 PM	6/11/2024 5:59:45 PM						
wslrelay.exe	24220	TCP	443	https	::1			::	SERVER	Listening					C:\Program Files\WSL\wslrelay.exe	Windows Subsystem for Linux	Windows Subsystem for Linux	2.1.5.0	Microsoft Corporation	6/10/2024 8:38:21 AM	SERVER\USER		A	6/11/2024 5:59:47 PM	6/11/2024 5:59:45 PM						
Unknown	0	TCP	443	https	192.168.1.26	58648		192.168.1.1		Time Wait										N/A				6/11/2024 6:04:43 PM							
Unknown	0	TCP	443	https	192.168.1.26	58649		192.168.1.1		Time Wait										N/A				6/11/2024 6:04:43 PM							
Unknown	0	TCP	443	https	192.168.1.26	58650		192.168.1.1		Time Wait										N/A				6/11/2024 6:04:43 PM							
Unknown	0	TCP	443	https	192.168.1.26	58651		192.168.1.1		Time Wait										N/A				6/11/2024 6:04:43 PM							
Unknown	0	TCP	443	https	192.168.1.26	58652		192.168.1.1		Time Wait										N/A				6/11/2024 6:04:43 PM							
Unknown	0	TCP	443	https	192.168.1.26	15035		192.168.1.1		Time Wait										N/A				6/11/2024 6:04:47 PM													
```

**Reply by szaimen** [reply, marked as answer]
> com.docker.backend.exe	1052	UDP	443	https	0.0.0.0										C:\Program Files\Docker\Docker\resources\com.docker.backend.exe	Docker Desktop Backend	Docker Desktop Backend	4.31.0.153195	Docker Inc.	6/10/2024 8:37:26 AM	SERVER\USER		A	6/11/2024 5:59:45 PM	6/11/2024 5:59:45 PM						
> com.docker.backend.exe	1052	UDP	443	https	::				SERVER						C:\Program Files\Docker\Docker\resources\com.docker.backend.exe	Docker Desktop Backend	Docker Desktop Backend	4.31.0.153195	Docker Inc.	6/10/2024 8:37:26 AM	SERVER\USER		A	6/11/2024 5:59:45 PM	6/11/2024 5:59:45 PM						
> com.docker.backend.exe	1052	TCP	443	https	0.0.0.0			0.0.0.0		Listening					C:\Program Files\Docker\Docker\resources\com.docker.backend.exe	Docker Desktop Backend	Docker Desktop Backend	4.31.0.153195	Docker Inc.	6/10/2024 8:37:26 AM	SERVER\USER		A	6/11/2024 5:59:47 PM	6/11/2024 5:59:45 PM						
> com.docker.backend.exe	1052	TCP	443	https	::			::	SERVER	Listening

This shows that the ports are opened correctly.



> {"level":"error","ts":1718021234.2177813,"logger":"tls.issuance.acme.acme_client","msg":"challenge failed","identifier":"xxx.xx.xxx.xx","challenge_type":"tls-alpn-01","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]}}
> {"level":"error","ts":1718021234.2178032,"logger":"tls.issuance.acme.acme_client","msg":"validating authorization","identifier":"xxx.xx.xxx.xx","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]},"order":"[https://acme-v02.api.letsencrypt.org/acme/order/1774073037/277116237427","attempt":1,"max_attempts":3}](https://acme-v02.api.letsencrypt.org/acme/order/1774073037/277116237427%22,%22attempt%22:1,%22max_attempts%22:3%7D)
> {"level":"error","ts":1718021234.2178178,"logger":"tls.obtain","msg":"could not get certificate from issuer","identifier":"xxx.xx.xxx.xx","issuer":"acme-v02.api.letsencrypt.org-directory","error":"HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)"}
> {"level":"error","ts":1718021234.2178354,"logger":"tls.obtain","msg":"will retry","error":"[xxx.xx.xxx.xx] Obtain: [xxx.xx.xxx.xx] solving challenge: xxx.xx.xxx.xx: [xxx.xx.xxx.xx] authorization failed: HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem) (ca=[https://acme-v02.api.letsencrypt.org/directory)","attempt":1,"retrying_in":60,"elapsed":11.04130646,"max_duration":2592000}](https://acme-v02.api.letsencrypt.org/directory)%22,%22attempt%22:1,%22retrying_in%22:60,%22elapsed%22:11.04130646,%22max_duration%22:2592000%7D)
> {"level":"error","ts":1718021305.2944057,"logger":"tls.issuance.acme.acme_client","msg":"challenge failed","identifier":"xxx.xx.xxx.xx","challenge_type":"tls-alpn-01","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]}}
> {"level":"error","ts":1718021305.2944527,"logger":"tls.issuance.acme.acme_client","msg":"validating authorization","identifier":"xxx.xx.xxx.xx","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]},"order":"[https://acme-staging-v02.api.letsencrypt.org/acme/order/151495713/17105190533","attempt":1,"max_attempts":3}](https://acme-staging-v02.api.letsencrypt.org/acme/order/151495713/17105190533%22,%22attempt%22:1,%22max_attempts%22:3%7D)
> {"level":"error","ts":1718021305.2944934,"logger":"tls.obtain","msg":"could not get certificate from issuer","identifier":"xxx.xx.xxx.xx","issuer":"acme-v02.api.letsencrypt.org-directory","error":"HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)"}
> {"level":"error","ts":1718021305.294508,"logger":"tls.obtain","msg":"will retry","error":"[xxx.xx.xxx.xx] Obtain: [xxx.xx.xxx.xx] solving challenge: xxx.xx.xxx.xx: [xxx.xx.xxx.xx] authorization failed: HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem) (ca=[https://acme-staging-v02.api.letsencrypt.org/directory)","attempt":2,"retrying_in":120,"elapsed":82.122759287,"max_duration":2592000}](https://acme-staging-v02.api.letsencrypt.org/directory)%22,%22attempt%22:2,%22retrying_in%22:120,%22elapsed%22:82.122759287,%22max_duration%22:2592000%7D)
> {"level":"error","ts":1718021436.1365576,"logger":"tls.issuance.acme.acme_client","msg":"challenge failed","identifier":"xxx.xx.xxx.xx","challenge_type":"tls-alpn-01","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]}}
> {"level":"error","ts":1718021436.1366057,"logger":"tls.issuance.acme.acme_client","msg":"validating authorization","identifier":"xxx.xx.xxx.xx","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)","instance":"","subproblems":[]},"order":"[https://acme-staging-v02.api.letsencrypt.org/acme/order/151495713/17105231643","attempt":1,"max_attempts":3}](https://acme-staging-v02.api.letsencrypt.org/acme/order/151495713/17105231643%22,%22attempt%22:1,%22max_attempts%22:3%7D)
> {"level":"error","ts":1718021436.1366274,"logger":"tls.obtain","msg":"could not get certificate from issuer","identifier":"xxx.xx.xxx.xx","issuer":"acme-v02.api.letsencrypt.org-directory","error":"HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem)"}
> {"level":"error","ts":1718021436.136653,"logger":"tls.obtain","msg":"will retry","error":"[xxx.xx.xxx.xx] Obtain: [xxx.xx.xxx.xx] solving challenge: xxx.xx.xxx.xx: [xxx.xx.xxx.xx] authorization failed: HTTP 400 urn:ietf:params:acme:error:connection - During secondary validation: xxx.xx.xxx.xx: Timeout during connect (likely firewall problem) (ca=[https://acme-staging-v02.api.letsencrypt.org/directory)","attempt":3,"retrying_in":120,"elapsed":212.972014677,"max_duration":2592000}](https://acme-staging-v02.api.letsencrypt.org/directory)%22,%22attempt%22:3,%22retrying_in%22:120,%22elapsed%22:212.972014677,%22max_duration%22:2592000%7D)

I still suspect a networking problem based on that logs...
